apiserver/facades/agent/uniter/mock_test.go

```diff
@@ -19,6 +19,7 @@ type fakeStorage struct {
 	storageInstance        func(names.StorageTag) (state.StorageInstance, error)
 	storageInstanceVolume  func(names.StorageTag) (state.Volume, error)
 	volumeAttachment       func(names.Tag, names.VolumeTag) (state.VolumeAttachment, error)
+	volumeAttachmentPlan   func(names.Tag, names.VolumeTag) (state.VolumeAttachmentPlan, error)
```

Comment:
I'm guessing that at this point there is too much history and other code involved to do this as a mockgen based mock. Is that your feeling?

Reply:
If I do the mockgen thing for this, I'd have to refactor for every other bit of test code here, which is something I do not want to do as part of this PR.